.coderabbit.yml

```diff
@@ -11,9 +11,9 @@ reviews:
   collapse_walkthrough: false
   poem: false
   auto_review:
-    enabled: true
+    enabled: false
     auto_incremental_review: false
-    drafts: false
+    drafts: true
```

Comment:
Why has this been changed?

Reply:
currently you can't request manual reviews on draft PRs, this allows that